sematic/cli/clean.py

```diff
@@ -0,0 +1,128 @@
+# Standard Library
+import logging
+import os
+from collections import Counter
+from logging.config import dictConfig
+from typing import List
+
+# Third-party
+import click
+
+# Sematic
+import sematic.api_client as api_client
+from sematic.api.endpoints.auth import get_cleaner_api_key
+from sematic.cli.cli import cli
+from sematic.config.config import switch_env
+from sematic.config.user_settings import UserSettingsVar
+from sematic.logging import make_log_config
+
+logger = logging.getLogger(__name__)
+
+
+@cli.command("clean", short_help="Clean up orphaned objects")
+@click.option(
+    "--orphaned-jobs",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Clean jobs whose associated runs have stopped.",
+)
+@click.option(
+    "--orphaned-resources",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help="Clean up external resources (ex: Ray clusters) whose runs have stopped.",
+)
+@click.option(
+    "--force",
+    is_flag=True,
+    show_default=True,
+    default=False,
+    help=(
+        "Mark the metadata for cleaned objects as being terminal "
+        "even if the underlying objects can't be verified as cleaned."
+    ),
+)
+def clean(orphaned_jobs: bool, orphaned_resources: bool, force: bool):
+    """
+    Clean up objects that are no longer needed.
+    """
+    running_as_cron_job = os.environ.get("RUNNING_AS_CRON_JOB", None) is not None
+    switch_env("user")
```

Comment:
How will the CLI invocation work on deployment if the "user" env is hardcoded?

Reply:
we'll configure env vars on the container it runs in so that it connects to the appropriate API